lib/fs-readdir-recursive.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module "fs-readdir-recursive" {
+  function read(
+    root: string,
+    filter?: (filename: string, index: number, dir: string) => boolean
+  ): string[];
+  export = read;
+}
```

Comment:
As a follow-up, could you open a PR to update https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/fs-readdir-recursive/index.d.ts so that then we can use `@types/fs-readdir-recursive`?

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/61344
Finish. I think we might be able to remove this package at some point.